minor grammatical fixes, can be ignored.

JacobMGEvans opened the pull request:
No major fixes can be ignored if preferred. 

1. Remove changed to removing 
2. Hyphen added JavaScript-powered 

Thank you for reading.

## Description
<!-- Please describe your changes -->

## How Has This Been Tested?
<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, tests ran to see how -->
<!-- your change affects other areas of the code, etc. -->

## Screenshots (jpeg or gifs if applicable):

## Types of changes
<!-- What types of changes does your code introduce?  -->
<!-- Bug fix (non-breaking change which fixes an issue) -->
<!-- New feature (non-breaking change which adds functionality) -->
<!-- Breaking change (fix or feature that would cause existing functionality to not work as expected) -->

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code has proper inline documentation.

JacobMGEvans committed:
minor grammatical fixes, can be ignored. 

No major fixes can be ignored if preferred. 

1. Remove changed to removing 
2. Hyphen added JavaScript-powered 

Thank you for reading.

ntwb commented on README.md:
```diff
@@ -22,25 +22,25 @@ Gutenberg looks at the editor as more than a content field, revisiting a layout
 Blocks are the unifying evolution of what is now covered, in different ways, by shortcodes, embeds, widgets, post formats, custom post types, theme options, meta-boxes, and other formatting elements. They embrace the breadth of functionality WordPress is capable of, with the clarity of a consistent user experience.
 
-Imagine a custom “employee” block that a client can drag to an About page to automatically display a picture, name, and bio. A whole universe of plugins that all extend WordPress in the same way. Simplified menus and widgets. Users who can instantly understand and use WordPress  -- and 90% of plugins. This will allow you to easily compose beautiful posts like <a href="http://moc.co/sandbox/exampl ...
+Imagine a custom “employee” block that a client can drag to an About page to automatically display a picture, name and bio. A whole universe of plugins that all extend WordPress in the same way. Simplified menus and widgets. Users who can instantly understand and use WordPress  -- and 90% of plugins. This will allow you to easily compose beautiful posts like <a href="http://moc.co/sandbox/example ...
```
WordPress uses [oxford commas](https://en.wikipedia.org/wiki/Oxford_Comma), we're going to need this comma back, please 😏

ntwb reviewed (changes requested):
Thanks @Cloud887, see above

JacobMGEvans committed:
update, oxford comma preferred.

> Oxford comma reinserted in a list of items greater than three. (personal preference is also Oxford comma =D)

Cloud887 commented:
Glad I could make some minor contribution to this great project. I also prefer the Oxford comma even given a convention choice. 

Thank you.

ntwb commented:
Thank you @Cloud887 for your contribution, every little bit helps 🥇

ntwb commented:
p.s https://github.com/WordPress/gutenberg/commit/0f6dd64e1e5f33d274659eeecf06f150330be8a7
